h2o-core/src/main/java/water/rapids/RadixCount.java

```diff
@@ -1,5 +1,11 @@
 package water.rapids;
 
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
My point is that if the TODO/FIXME is in the code, there is no owner, no plan to implement it. 
So at the end it is useless information. However, having it in issue tracker seems more beneficial, since we can use the issue as a starter tasks for new H2Odoers or internships.